src/reporters.js

```diff
@@ -73,7 +73,7 @@ function repoToConsoleLine(repo) {
             if (metric.info.name === metricNames[i]) {
                 if (metric.result === null) {
                     logger.err("error on table");
-                } else if (metric.result.result === undefined) {
+                } else if (metric.result == undefined || metric.result.result === undefined) {
```

Comment:
Can we change this to `else if (!_.isObject(metric.result) || !_.isObject(metric.result.result))`?

Reply:
yea, sure